src/test/java/ru/training/at/hw3/main/differentelementspage/DifferentElementsPage.java

```diff
@@ -0,0 +1,60 @@
+package ru.training.at.hw3.main.differentelementspage;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+public class DifferentElementsPage {
+    WebDriver driver;
+    WebDriverWait wait;
+
+    @FindBy(xpath = "//label[1]/input")
+    WebElement waterCheckBox;
+
+    @FindBy(xpath = "//label[3]/input")
+    WebElement windCheckBox;
+
+    @FindBy(xpath = "//label[4]/input[@name='metal']")
+    WebElement selenRadio;
+
+    @FindBy(xpath = "//div[@class='colors']")
+    WebElement colorsMenu;
```

Comment:
You could interact with select menu with special Selenium class - Select. It will be more flexible, especially if you create a method which will be able to work with all elements in dropdown by method parameter.